GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -85,6 +85,7 @@
     public static final String DURATION_COLUMN_KEY = "surveyalTime";
     
     public static final String METADATA_HEADER = "Metadata";
+    public static final String NEW_DATA_PATTERN = "^new-\\d+"; // new- followed by one or more digits
```

Comment:
Where is this used? i can't seem to locate it..

Reply:
Fixed.